### DISCUSSION TITLE
Further issues with importing

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#809] [Q&A] [answered]
```Solidity
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";
"@chainlink/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol";
```

I am still having no luck trying to import this way. It constantly comes with a error and cant compile, due to not found. This is in VS code and i know that youre supposed to pass through github, can someone explain how to pass through each file in the config.yaml.

i have been having this issues since Lesson 3 and am now still struggling at lesson 6. I have tried flattening the code as well and this just ends up giving long run errors

**Comment by ghost** [comment]
CompilerError: solc returned the following errors:

```BASH
contracts/tests/FundMe.sol:8:1: ParserError: Source "smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
^--------------------------------------------------------------------------^

contracts/tests/FundMe.sol:9:1: ParserError: Source "smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/vendor/SafeMathChainlink.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";
^------------------------------------------------------------------^

contracts/tests/MockV3Aggregator.sol:4:1: ParserError: Source "smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol";
^----------------------------------------------------------------------------^

Matts-MacBook-Air:brownie_fund_me mattwilson$ 
```

**Comment by cromewar** [comment, marked as answer]
Hello @mattwilsonbmx Could you please share your brownie-config.yaml file?

**Comment by amicable-alpaca** [comment]
I'm having a similar error, here is my yaml file. Thanks for the help

![image](https://user-images.githubusercontent.com/98236132/150702736-ccc6cd42-60bf-453f-91b2-0242fab008a9.png)

**Reply by cromewar** [reply]
Please delete the build folder and compile again, if that don't work, go to your home and delete the `.brownie` folder and try compiling again.

**Reply by amicable-alpaca** [reply]
What do you mean when you say "go to your home and delete the .brownie folder" ?

Do you mean my "Home" folder? If so it is not there. I am using Ubuntu and a virtual environment, btw.

Where can I find the .brownie folder?

Thanks for the help.

**Reply by cromewar** [reply]
The Home Folder is on your primary location, `~/` as you are on ubuntu you'll see also the documents, images and other folders. .brownie folder is a hidden one, to see it press `Ctrl + .` then you'll see all the hidden files, there delete the .brownie one.

**Reply by amicable-alpaca** [reply]
I was able to resolve it, thanks! Sorry for the delayed response!
